tests/unit/test_server_mock_conductor.js

```diff
@@ -372,10 +437,8 @@ describe("Server with mock Conductor", () => {
     expect(res2).to.deep.equal({
       type: "error",
       payload: {
-        source: "HoloError",
-        error: "HoloError",
-        message: "Failed during Conductor AppInfo call",
-        stack: []
+        "error": "Error",
```

Comment:
Why are we getting back a message that is thrown after being `hhdt parsed` now?

Reply:
https://github.com/Holo-Host/chaperone/pull/106/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R1010